pkg/kubectl/resource/builder.go

```diff
@@ -806,12 +806,13 @@ func HasNames(args []string) (bool, error) {
 
 // MultipleTypesRequested returns true if the provided args contain multiple resource kinds
 func MultipleTypesRequested(args []string) bool {
+	if len(args) == 1 && args[0] == "all" {
```

Comment:
They're only the set passed to the sub command (in `kubectl get all`, `[all]` would be the args slice)